python/ray/util/BUILD

```diff
@@ -6,3 +6,12 @@ doctest(
     files = glob(["*.py"], exclude=["client_connect.py", "iter.py", "iter_metrics.py"]),
     tags = ["team:core"]
 )
+
```

Comment:
Add `load("//bazel:python.bzl", "py_test_module_list")` to the top of the file to fix build failures.